components/Navbar.js

```diff
@@ -45,6 +45,13 @@ export default function Navbar() {
                   <li><ActiveLink activeClassName="active" href="/committees#hack" passHref={true}><button type="button" onClick={menuActivate}>Hack</button></ActiveLink></li>
                 </ul>
               </li>
+              <li><button>initiatives</button></li>
```

Comment:
Since this isn't clickable, can we use a different element instead of `<button>`? This will trip up a screenreader/makes our nav not as accessible.

Reply:
No problem. Now, there is just the <li>. Do you think replacing by another element is accurate ?

Btw, "npm run lint-fix" shows several errors but not relating to my work i think.

Reply:
> No problem. Now, there is just the li. Do you think replacing it by another element is accurate ?

Perfect, seems good to me!

> Btw, "npm run lint-fix" shows several errors but not relating to my work I think.

Interesting, when I run `npm run lint` off of `main`, I have no problems. This also passes CI, so I wouldn't overstress!